include/revng/Model/Binary.h

```diff
@@ -615,17 +38,8 @@ class model::Binary {
   bool verify(VerifyHelper &VH) const;
   void dump() const debug_function;
 };
-INTROSPECTION_NS(model,
-                 Binary,
-                 Functions,
-                 ImportedDynamicFunctions,
-                 Types,
-                 Architecture,
-                 Segments)
 
-template<>
-struct llvm::yaml::MappingTraits<model::Binary>
-  : public TupleLikeMappingTraits<model::Binary> {};
+#include "revng/Model/GeneratedBinary_late.h"
```

Comment:
Can we push this in the generated file then?

Reply:
The `IsYamlizable` assert can be moved to the autogenerated file, but
```
static_assert(validateTupleTree<model::Binary>(OnlyKOC),
              "Only SortedVectors and MutableSets with YAMLizable keys are "
              "allowed");
```
cannot, because `OnlyKOC` does not apply to all classes. I can add an option to the model YAML, but it's not ideal, what do you think?